backend/app/main.py

```diff
@@ -41,11 +41,12 @@
     Returns:
         The generated OAuth token information
     """
-    user = crud.check_user(db, form_data.username, form_data.password)
+    # form_data.username will need to be the email the user signed up with. user_email is a str
+    user = crud.check_user(db, form_data.username, form_data.password)  # Shadows name user from outer scope
```

Comment:
I don't understand the comment? 😕

Reply:
It was like 4am I wasn't thinking straight lmao

Reply:
Comment is still there?